dev.fredol.open-tv.yaml

```
@@ -0,0 +1,1152 @@
+id: dev.fredol.open-tv
+
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+
+command: open_tv
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --filesystem=host:ro
+  - --socket=pulseaudio
+  - --filesystem=/tmp:rw
+  - --filesystem=xdg-pictures:rw
+  - --filesystem=xdg-videos:rw
+  - --filesystem=home/.var/app/io.mpv.Mpv/config/mpv/scripts:create
+  - --env=LC_NUMERIC=C
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+  - --filesystem=xdg-run/pipewire-0:ro
+  - --own-name=org.mpris.MediaPlayer2.mpv.*
+  - --talk-name=org.gnome.SessionManager
+
+cleanup:
+  - '*.la'
+  - '*.a'
+
+environment:
+  WEBKIT_DISABLE_DMABUF_RENDERER: 1
+
+modules:
+  - name: binary
```

```suggestion
  - name: opentv
```

and since this is the main module it should be last in the list.